sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManager.scala

```diff
@@ -203,7 +204,12 @@ abstract class RapidsShuffleInternalManagerBase(conf: SparkConf, isDriver: Boole
       logWarning("Rapids Shuffle Plugin is falling back to SortShuffleManager because " +
         "external shuffle is enabled")
     }
-    fallThroughDueToExternalShuffle
+    val isAdaptiveEnabled = conf.get(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key, "false").toBoolean
```

Comment:
There is a `stringDefault` value in `SQLConf.ADAPTIVE_EXECUTION_ENABLED` which should let us stay up to date with the default if it changes.  The main problem with that is that we are compiling this against one specific version of spark so if the default changes from one version to another this will not stay up to date.

Reply:
This is something that we could potentially do in the shim layer though so that it does compile against each version we support. I'll look into doing that.

Reply:
@andygrove we should wait on this PR then? Or you are thinking a different PR?

Reply:
I'll add it to this PR.

Reply:
Actually, this is a boolean config so it has to have a value and it doesn't make sense to check for a default value. I'll remove the default part instead.

Reply:
This is updated now.